.gitignore

```diff
@@ -1,2 +1,3 @@
 .DS_Store
 *.pyc
+/logs
```

Comment:
Are logs stored at mentii/backend/logs or at mentii/logs. If its in backend/logs we should check to make sure they are not publicly visible via a url since they would be within the webserver's space